javascript-if/main.js

```diff
@@ -0,0 +1,87 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return ('true');
```

Comment:
There is nothing wrong with wrapping your return values with (), but just to confirm it is not required. 

```javascript
return 'true'
```

Reply:
Thank you